catalog/publishers.jsonld

```diff
@@ -3,27 +3,33 @@
   "@graph": [
     {
       "@id": "http://www.getty.edu/research/",
-      "identifier": "Getty"
+      "@type": "Organization",
+      "alternateName": "Getty"
```

Comment:
Would it be an option to also use the property `name` for each creator (e.g. "Getty Research Institute")? Otherwise it may not be clear what the alternate name is an alias for?